exercises/exercise-list.md

```diff
@@ -0,0 +1,278 @@
+# Java For Beginners - Exercises
+
+* [Topic 2 Exercises](#exercise-list-topic-2)
+* [Topic 3 Exercises](#exercise-list-topic-3)
+
+# Exercise List Topic 2
+
+This exercise list is to practice the content presented in [lecture 2](../lessons/lesson2)
+
```

Comment:
I think we should add some instructions here about the unique codes and what they refer to within the code repo/ how to use them. Maybe give an example for the first one "for instance the Variables and Basic Operators exercise has unique code VABO. So you can start this exercise off by finding folder xxxx" or something like that